config.rb

```diff
@@ -66,6 +66,12 @@
 # If you want Middleman to listen on a different port, you can set that below
 set :port, 4567
 
+configure :development do
+  # during development, redirect the index to the docs page because that's what
+  # we're most interested in testing
+  redirect "index.html", to: "docs/index.html.md"
```

Comment:
Is there documentation in ruby for this `redirect` call? If so, can you comment a link so I can read a bit about it. I am see `redirect_to` documentation for ruby but not `redirect`